### PR TITLE
Track2 sdk:add update function for account client

### PR DESCRIPTION
**Pull request by MartinForReal**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Track2 sdk:add update function for account client
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/azclient/accountclient/custom.go [hunk 1]
```diff
@@ -38,6 +38,17 @@ func (client *Client) Create(ctx context.Context, resourceGroupName string, reso
 	return nil, nil
 }
 
+func (client *Client) Update(ctx context.Context, resourceGroupName string, resourceName string, parameters *armstorage.AccountUpdateParameters) (*armstorage.Account, error) {
+	if parameters == nil {
+		parameters = &armstorage.AccountUpdateParameters{}
```
Comment (zarvd) [inline pkg/azclient/accountclient/custom.go:43]: Just curious, what will happen when updating with nil parameters?

**Comment by nilo19** [comment]
is it possible to cover negative cases in track2 unit tests?

File: pkg/azclient/accountclient/custom.go (hunk 1 above)
Comment (MartinForReal) [reply pkg/azclient/accountclient/custom.go:43]: nothing changed.

**Pull request merged**